Source/UIButton+Utils.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  UIButton+Utils.swift
+//  Lipstick
+//
+//  Created by Tadeas Kriz on 31/03/16.
+//  Copyright © 2016 Brightify. All rights reserved.
+//
+
+import UIKit
+
+extension UIButton {
+    
+    public convenience init(title: String) {
+        self.init()
+        
+        setTitle(title, for: UIControlState())
+    }
+}
+
+extension UIButton {
+    
+    public func setBackgroundColor(_ color: UIColor, forState state: UIControlState) {
```

Comment:
Should be `setBackgroundColor(_ color: UIColor, for state: UIControlState)` to be consistent with UIKit.